### ISSUE TITLE
Images missing alt

### ISSUE THREAD

**Issue by FestiveKyle** [canada-ca/tracker#2634]
All images need to have alt text (whether actually descriptive or "" to be skipped). This is missing on many of the svgs. Any "eye-candy" image can just use alt="" if aren't adding context to their area.

**Comment by AdamPrins** [comment]
Are there any areas that still need additional labels?
Most were addressed in #2705
